ocaml/lambda/lambda.mli

```diff
@@ -334,6 +334,41 @@ type primitive =
      handlers, finalizers, memprof callbacks, etc, as well as GCs and
      GC slices, so should not be moved or optimised away. *)
   | Ppoll
+    (* In-place reuse of heap blocks.
+     Each of these takes the allocation to be reused as first argument.
+     The following arguments correspond to the fields that are Reuse_set. *)
+  | Preuseblock of
```

Comment:
Oh, right, that is the intended meaning. If `Preuseblock` was a function, we should have: `Preuseblock : memory cell -> overwritten args -> new block`. You should not refer to the overwritten memory cell after it was passed to `Preuseblock`. In fact, referring to the overwritten memory cell again would break lots of properties since the overwrite may change its type.